contrib/automation_tests/orbit_manual_instrumentation_silenus.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.capture_window import Capture, CheckTimers, CollapseTrack, VerifyTracksExist
+from test_cases.symbols_tab import LoadSymbols
+"""Test Orbit's manual instrumentation on Silenus.
+
+Before this script is run there needs to be a gamelet reserved and the
+version of "triangle.exe" with manual instrumentation has to be started.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test, it needs
+to be run from 64 bit python.
+
+This automation script covers the following workflow:
+ - start Orbit
```

Comment:
`- start Orbit` is probably copy pasted from another test. Just remove, none of the tests is actually doing this.

Reply:
Definitely from copy-pasting, I removed "start Orbit" from all scripts.